packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -32,6 +38,22 @@ export class HttpForwarder
     };
   }
 
+  private updateHostHeaders(baseUrl: string, headers?: IHttpNameValue) {
+    // no headers? do nothing
+    if (!headers) return headers;
+
+    // host header provided? override with actual hostname
+    if (headers.hasOwnProperty('host')) {
+      return {
+        ...headers,
+        host: urijs(baseUrl).host(),
```

Comment:
You can avoid installing a new library and just use builtin `URI`
```suggestion
        host: (new URL(baseUrl)).host(),
```

Reply:
That's what I was looking for, happy to remove urijs dependency :)